client/src/shared/LoginInputHandler.tsx

```diff
@@ -0,0 +1,27 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+
+export const passwordRegExp =
```

Comment:
What do you think about moving passwordRegExp into some separate file called eg. constants.ts and keep it there because it is used in several files?

Reply:
Good idea, I will move it :)

Reply:
Definitely sth went wrong.. :D I will fix it